CSVSource/src/main/java/io/vantiq/extsrc/CSVSource/CSV.java

```diff
@@ -616,10 +617,10 @@ void handleExistingFiles(String fileFolderPath) {
             String fullFilePath = path.toString() + File.separator + fileStr;
             File file = new File(fullFilePath);
             if (file.exists()) {
-                try (//FileWriter fw = new FileWriter(file, true); 
-//                BufferedWriter bw = new BufferedWriter(fw)) 
-                BufferedWriter bw = new BufferedWriter( new OutputStreamWriter(new FileOutputStream(fullFilePath), "UTF-8"))) 
-                {
+                try (// FileWriter fw = new FileWriter(file, true);
+                        // BufferedWriter bw = new BufferedWriter(fw))
```

Comment:
Please remove all this old code.  We all leave it in to make it easier to revert, but please remove before PR time.